packages/python-google-compute-engine/google_compute_engine/boto/boto_config.py

```diff
@@ -58,22 +58,23 @@ def __init__(self, project_id=None, debug=False):
     self.watcher = metadata_watcher.MetadataWatcher(logger=self.logger)
     self._CreateConfig(project_id)
 
-  def _GetNumericProjectId(self):
-    """Get the numeric project ID for this VM.
+  def _GetProjectId(self):
+    """Get the (non-numeric) project ID for this VM.
 
     Returns:
-      string, the numeric project ID if one is found.
+      string, the project ID if one is found.
     """
-    project_id = 'project/numeric-project-id'
-    return self.watcher.GetMetadata(metadata_key=project_id, recursive=False)
+    project_id_key = 'project/project-id'
+    return self.watcher.GetMetadata(metadata_key=project_id_key,
```

Comment:
Please line wrap from the first open paren.

Reply:
Done.